backtrace-library/src/main/java/backtraceio/library/breadcrumbs/BacktraceQueueFileHelper.java

```diff
@@ -0,0 +1,77 @@
+package backtraceio.library.breadcrumbs;
+
+import com.squareup.tape.QueueFile;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.nio.charset.StandardCharsets;
+
+import backtraceio.library.logger.BacktraceLogger;
+
+public class BacktraceQueueFileHelper {
+    /**
+     * The base directory of the breadcrumb logs
+     */
+    private String breadcrumbLogDirectory;
+
+    /**
+     * The breadcrumb storage file
+     */
+    private QueueFile breadcrumbStore;
+
+    private String logFileName = "bt-breadcrumbs-0";
+
+    private final String LOG_TAG = BacktraceBreadcrumbs.class.getSimpleName();
+
+    private int maxQueueFileSizeBytes;
+
+    private Method usedBytes;
+
+    public BacktraceQueueFileHelper(String breadcrumbLogDirectory, int maxQueueFileSizeBytes) throws IOException, NoSuchMethodException {
+        this.breadcrumbLogDirectory = breadcrumbLogDirectory;
+        File breadcrumbLogsDir = new File(breadcrumbLogDirectory);
+        breadcrumbLogsDir.mkdir();
+        breadcrumbStore = new QueueFile(new File(breadcrumbLogsDir + "/" + logFileName));
+
+        usedBytes = QueueFile.class.getDeclaredMethod("usedBytes");
+        usedBytes.setAccessible(true);
+
+        // This minimum file size comes from QueueFile::INITIAL_LENGTH
+        if (maxQueueFileSizeBytes < 4096) {
```

Comment:
place save `4096` to private final variable with self explaining name, and maybe comment why such value

Reply:
Will do, thank you!